client/my-sites/checkout/checkout/index.jsx

```diff
@@ -361,6 +363,10 @@ export class Checkout extends React.Component {
 			0
 		);
 
+		if ( redirectTo && ! isExternal( redirectTo ) ) {
```

Comment:
Adding the `if` condition here will not process the GSuite or Concierge Session upsells that follow in the rest of the function. 

**Testing instructions:**

**Without applying the PR**
_Concierge upsell_
1. Add a plan to cart (don't add a custom domain to cart)
2. Proceed to checkout and purchase plan
3. Verify that you are redirect to the Concierge Session upsell at `/checkout/{SITE_SLUG}/add-support-session` or `/checkout/{SITE_SLUG}/add-quickstart-session`(depends on which control group it puts you in since these two URLs are being abtested).

_GSuite upsell_
Repeat steps 1 to 3 by above by adding a plan _and_ domain to cart. Verify that after checkout, you're redirected to the GSuite upsell at `/checkout/{SITE_SLUG}/with-gsuite/`

**After applying the PR**
Repeat steps 1 to 4 above. Post purchase, we're redirect to `/me` without showing the Concierge or GSuite upsell nudges.

Reply:
Another issue is that the GSuite and Concierge Upsell pages have a `Skip` button(check screenshots https://cloudup.com/cquYtIHa5l7). These need to adhere to the redirect link too.

Reply:
Hmmm. I wonder if we need a more comprehensive routing solution. At the moment its a terrible mess.